setup.py

```diff
@@ -17,14 +17,16 @@
           'distributed',
           'toolz',
           'pytest',		
-          'fiona',
-          'cartopy',	
-          'shapely',
-          'seaborn',		  
           'matplotlib',
           'geopandas',
           'pandas',
-          'hs_restclient',
+          'hs_restclient'
           ],
+       extras_require={'pysumma': [
```

Comment:
I think it might be better to call this something like `spatial` or `plotting` so that when you install you specify `pip install pysumma[spatial]` instead of `pip install pysumma[pysumma]` which is confusing.